client-v2/src/util/collectionUtils.ts

```diff
@@ -2,11 +2,26 @@ import React from 'react';
 import { ThunkResult, Dispatch } from 'types/Store';
 import { PosSpec } from 'lib/dnd';
 import { insertArticleFragment } from 'actions/ArticleFragments';
+import { CapiArticle } from 'types/Capi';
 
-const dropToArticle = (e: React.DragEvent): string | null => {
+export interface RefDrop {
+  type: 'REF';
```

Comment:
What does 'REF' mean? (I have it mentally pegged as 'anything that hasn't come from the feed')

Reply:
👍